extensions/positron-assistant/src/extension.ts

```diff
@@ -6,7 +6,7 @@
 import * as vscode from 'vscode';
 import * as positron from 'positron';
 import { EncryptedSecretStorage, expandConfigToSource, getEnabledProviders, getModelConfiguration, getModelConfigurations, getStoredModels, GlobalSecretStorage, logStoredModels, ModelConfig, SecretStorage, showConfigurationDialog, StoredModelConfig } from './config';
-import { availableModels, newLanguageModel } from './models';
+import { availableModels, getLanguageModels, createModelConfigFromEnv, newLanguageModel } from './models';
```

Comment:
Nit: remove unused import
```suggestion
import { availableModels, createModelConfigFromEnv, newLanguageModel } from './models';
```